components/pages/Homepage/EventCard.tsx

```diff
@@ -75,7 +75,7 @@ const Category = ({ category }: CategoryProps) => {
 
 export const EventCard = ({ event }: EventCardProps) => {
   return (
-    <div className={styles["event-card"]}>
+    <div className={`${styles["event-card"]} eventcard`}>
```

Comment:
Rather than adding a redundant classname here, I'd add `data-cy`: 

```suggestion
    <div className={`${styles["event-card"]}`} data-cy="event-card">
```